### PR TITLE
Add variable support

### PR DESCRIPTION
**Pull request by mjablecnik**
## Description

Hello I added support variables for colors and units. https://github.com/schultek/jaspr/issues/279
Right now you can define variables in your html:
```
      .raw({
        '--background-color': '#fff',
        '--text-color': Colors.blue.value,
        '--text-size': 14.pt.value,
        // other values...
      }),

```
and use it as: `Color.variable('--text-color')` or `Unit.variable('--text-size')`

## Type of Change

✨ New feature or improvement

## Ready Checklist

- [ ] I've read the [Contribution Guide](https://github.com/schultek/jaspr/blob/main/CONTRIBUTING.md).
- [x] In case this PR changes one of the core packages, I've modified the respective **CHANGELOG.md** file using
      the [semantic_changelog](https://github.com/rrousselGit/semantic_changelog) format.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added myself to the AUTHORS file (optional, if you want to).

### CODE REVIEW

**Comment by schultek** [comment]
Nice change. Can you also add this for `Angle`, then it should cover probably >90% of all uses of variables.

And what do you think about calling it `.var()` instead of `.variable()`?

**Review by schultek — COMMENTED** [review]
Nice change. Can you also add this for Angle, then it should cover probably >90% of all uses of variables.

And what do you think about calling it .var() instead of .variable()?

Dont forget to update the changelog

**Comment by mjablecnik** [comment]
Hello, I added variable also for Angle and updated changelog.
I also wanted to call it as .var() instead of .variable() but there is problem because `var` is Dart key word:

Screenshot 2024-09-09 at 0 09 43
vs
Screenshot 2024-09-09 at 0 09 24
or
Screenshot 2024-09-09 at 0 15 00

So we must name it somehow else then only .var()

File: packages/jaspr/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 0.16.0
```
Comment (schultek) [inline packages/jaspr/CHANGELOG.md]:
```suggestion
## Unreleased minor
```

**Comment by mjablecnik** [comment]
@schultek Is everything ok? Can you already merge it?

**Review by schultek — APPROVED** [review]
LGTM

**Pull request merged**